nlp_apps.ipynb

```diff
@@ -371,6 +372,410 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can start predicting the disputed papers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Paper No. 49\n",
+      "Hamilton: 0.18218476722264856\n",
+      "Madison : 0.8178151126501306\n",
+      "Jay     : 1.2012722099721584e-07\n",
+      "----------------------\n",
+      "Paper No. 50\n",
+      "Hamilton: 0.006340777113564324\n",
+      "Madison : 0.9935600714606485\n",
+      "Jay     : 9.915142578703363e-05\n",
+      "----------------------\n",
+      "Paper No. 51\n",
+      "Hamilton: 0.10807398451170964\n",
+      "Madison : 0.8919260093780947\n",
+      "Jay     : 6.11019566801153e-09\n",
+      "----------------------\n",
+      "Paper No. 52\n",
+      "Hamilton: 0.015755507847563528\n",
+      "Madison : 0.9842245750173423\n",
+      "Jay     : 1.9917135094100632e-05\n",
+      "----------------------\n",
+      "Paper No. 53\n",
+      "Hamilton: 0.16148149622286845\n",
+      "Madison : 0.8385181396174793\n",
+      "Jay     : 3.641596521788814e-07\n",
+      "----------------------\n",
+      "Paper No. 54\n",
+      "Hamilton: 0.1202445807489968\n",
+      "Madison : 0.8797554191935693\n",
+      "Jay     : 5.743394071176045e-11\n",
+      "----------------------\n",
+      "Paper No. 55\n",
+      "Hamilton: 0.10014174623125195\n",
+      "Madison : 0.8998582478040609\n",
+      "Jay     : 5.964687179083329e-09\n",
+      "----------------------\n",
+      "Paper No. 56\n",
+      "Hamilton: 0.15930217913525455\n",
+      "Madison : 0.8406948696158869\n",
+      "Jay     : 2.9512488585096405e-06\n",
+      "----------------------\n",
+      "Paper No. 57\n",
+      "Hamilton: 0.3106575736716812\n",
+      "Madison : 0.6893423580295986\n",
+      "Jay     : 6.829872019646261e-08\n",
+      "----------------------\n",
+      "Paper No. 58\n",
+      "Hamilton: 0.08144023779669217\n",
+      "Madison : 0.9185597621646735\n",
+      "Jay     : 3.8634360540381284e-11\n",
+      "----------------------\n",
+      "Paper No. 18\n",
+      "Hamilton: 7.762932414823314e-06\n",
+      "Madison : 0.5114716240007965\n",
+      "Jay     : 0.4885206130667886\n",
+      "----------------------\n",
+      "Paper No. 19\n",
+      "Hamilton: 0.011570316420346522\n",
+      "Madison : 0.5281730401297515\n",
+      "Jay     : 0.4602566434499019\n",
+      "----------------------\n",
+      "Paper No. 20\n",
+      "Hamilton: 0.14651509965391551\n",
+      "Madison : 0.5342142523806944\n",
+      "Jay     : 0.31927064796538995\n",
+      "----------------------\n",
+      "Paper No. 64\n",
+      "Hamilton: 0.5756065218890194\n",
+      "Madison : 0.3648418106830272\n",
+      "Jay     : 0.059551667427953384\n",
+      "----------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "for d in disputed:\n",
```

Comment:
I would probably do this something like

     for d in disputed:
         results = ['{}: {:.2f}'.format(name, probs[(name, 1)]) for name in 'Hamilton Madison Jay'.split()]
         print('Paper No. {}: {}'.format(d, ' '.join(results)